lambda_functions/README.md

```diff
@@ -1,16 +1,21 @@
 # Healthcheck Lambdas
 
-The lambdas for the SML Portal healthcheck can be changed to do this you will need to follow some steps after making your code changes.
+The lambdas for the SML Portal healthcheck require manual zipping.
+
+This is due to their runtime version and their need for dependencies like request.
+
+You can complete the upload using the below guide.
 
 ## Updating healthcheck.zip
 
 Run the following commands to update the healthcheck.zip (starting in the lambda_functions/healthcheck directory)
 
 ```
     mkdir packages
-    pip install --target ./packages requests
-    pip install --target ./packages boto3
-    pip install --target ./packages <any packages you need>
+    cd packages
```

Comment:
Shouldn't this be part of the CICD build though? That way you don't need someone to manually do it?

Reply:
I can look into this :)

Reply:
sorted